.github/workflows/gradle-check.yml

```
@@ -31,8 +27,17 @@ jobs:
         with:
           ref: ${{ github.event.pull_request.head.sha }}
 
+      - name: Get changed files
+        id: changed-files-specific
+        uses: tj-actions/changed-files@v44
+        with:
+          files_ignore: |
+            release-notes/**
```

I don't trust the `**` annotation, lets be very selective
```suggestion
            release-notes/*.md
```